.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,39 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG] in GeoHealthCheck (replace with your title}"
```

Comment:
Do we need `[BUG]` in the title?  We usually label as a bug as part triaging?

Reply:
Ok, fixed.